index.html

```diff
@@ -7,57 +7,215 @@
+                                    no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li class="list1">HTML</li>
+                            <li class="list2">Bootstrap</li>
+                            <li class="list3">Ruby</li>
+                        </ul>
+                        <button type="button">See project</button>
+                    </article>
+                    <article class="card">
+                        <h5>Profesional Art Printing Data</h5>
+                            <p>A daily selection of privately personalized reads; 
+                                    no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li class="list1">HTML</li>
+                            <li class="list2">Bootstrap</li>
+                            <li class="list3">Ruby</li>
+                        </ul>
+                        <button type="button">See project</button>
+                    </article>
+                    <article class="card">
+                        <h5>Profesional Art Printing Data</h5>
+                            <p>A daily selection of privately personalized reads; 
+                                    no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li class="list1">HTML</li>
+                            <li class="list2">Bootstrap</li>
+                            <li class="list3">Ruby</li>
+                        </ul>
+                        <button type="button">See project</button>
+                    </article>
+                    <article class="card">
+                        <h5>Profesional Art Printing Data</h5>
+                            <p>A daily selection of privately personalized reads; 
+                                    no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul>
+                            <li class="list1">HTML</li>
+                            <li class="list2">Bootstrap</li>
+                            <li class="list3">Ruby</li>
+                        </ul>
+                        <button type="button">See project</button>
+                    </article>
+            </section>
+            <!-- about section -->
+            <section class="about">
+                <h2>About me</h2>
+                    <p>Hi..!! I’m a web developer! I can help you build a product, feature or website. Look 
+                                through some of my work and experience! If you like what 
+                                you see and have a project you need coded,
+                                don't hesitate to contact me.
+                </p>
```

Comment:
- [ ] You can also increase the [letter-spacing](https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing) and [line height](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height) properties of this paragraph a bit. To ensure you have a pixel-perfect design.

- Your design.
<img width="150" alt="wrong" src="https://user-images.githubusercontent.com/41163571/227706189-68f490b9-6cd3-40f6-8bcd-b9bd5ae2a2a4.png">

- Figma design
<img width="308" alt="right" src="https://user-images.githubusercontent.com/41163571/227706178-4296cdcc-8d68-4e1b-8a05-bf82af018741.png">

Reply:
Letter spacing and line height adjusted.